### ISSUE TITLE
[Feature]: Can recognize local lyrics files

### ISSUE THREAD

**Issue by oliver-xie666** [fast4x/RiMusic#1862]
### Description

my Android devices have some  lyrics files with the songs，i hope Can recognize local lyrics files
 

### Suggested Solution

my Android devices have some  lyrics files with the songs，i hope Can recognize local lyrics files

**Comment by aneesh1122** [comment]
Duplicate of #1816

**Comment by ikanakova** [comment]
@kris20030705 I think fast4x cares. He has effective time management. If you feel that your issue is of the highest priority (and therefore feel the need to post about it even in an issue unrelated to it), you'll have to fix it yourself right away or, like others, wait patiently.

**Comment by aneesh1122** [comment]
> Didn't you face deleteing songs from local storage but still showing in RiMusic..!? I already mentioned that but they don't care.

Nope. Never did. btw @fast4x isn't getting paid to solve your 'bugs' so show some respect. You can always fork this, solve your bugs and publish the bug-less version otherwise just create the issue and do your thing.
